content/pages/coc.rst

```diff
@@ -0,0 +1,88 @@
+Code of Conduct
+===============
+
+Pyninsula is dedicated to providing a harassment-free conference experience
+for everyone, regardless of gender, gender identity and expression,
+sexual orientation, disability, physical appearance, body size, race,
+age or religion.
+We do not tolerate harassment of conference participants in any form.
+Sexual language and imagery is not appropriate in the meetup,
+including talks.
+Pyninsula participants violating these rules may be sanctioned or
+expelled from the venue at the discretion of the meetup organizers.
+
+Harassment includes, but is not limited to:
+
+* Offensive verbal comments related to gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age, or religion.
+* Sexual images in public spaces
+* Deliberate intimidation, stalking, or following 
+* Harassing photography or recording
+* Sustained disruption of talks or other events
+* Inappropriate physical contact
+* Unwelcome sexual attention
+* Advocating for, or encouraging, any of the above behaviour
+
+Enforcement
```

Comment:
one thing that's occurred to me is: should we be worried about retaliation?  suppose A and B are at meetup 1, and A intimidates B, which B then reports to us.  at meetup 2 we announce a policy designed to address that intimidation, and A and B are both present.  do we worry about A retaliating against B?

i don't have an answer, by the way - it's just that if i were B in the above scenario, i might get sweaty palms if i saw A in the crowd and then heard the policy announcement.  having an announcement on the site about the CoC and updated policies and announcing during the meetup people read it seems like a way to inform attendees that they need to listen up without creating such a situation.  but it may not be a strong enough communication.

i might also just be overthinking this.